src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,36 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  (
+    <div className="PostInfo">
+      <div className="postInfo__header">
+        <h3 className="PostInfo__title">
+          {post.title}
```

Comment:
you can also destructure your `post` properties

Reply:
agree, that definitely makes sense to me